opensearch-operator/controllers/opensearchController.go

```diff
@@ -113,7 +114,7 @@ func (r *OpenSearchClusterReconciler) Reconcile(ctx context.Context, req ctrl.Re
 				// so that it can be retried
 				return result, err
 			}
-
+			r.Recorder.Event(r.Instance, "Normal", "Cluster", fmt.Sprintf("Deleting cluster %s/%s", r.Instance.Namespace, r.Instance.Name))
```

Comment:
lets make every start finish fail message with the same prefix.
start logic message start with
Started to ...
failed logic
Failed to
finish logic
Finished to...